### PR TITLE
Feature/dynamic django runner

### PR DESCRIPTION
**Pull request by rodrigondec**
related to #122

### CODE REVIEW

**Comment by rodrigondec** [comment]
Update?

File: behave_django/runner.py [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+from django.test.utils import get_runner
 
 from behave_django.environment import BehaveHooksMixin
 from behave_django.testcase import (BehaviorDrivenTestCase,
                                     ExistingDatabaseTestCase,
                                     DjangoSimpleTestCase)
 
 
-class BehaviorDrivenTestRunner(DiscoverRunner, BehaveHooksMixin):
+DJANGO_CONFIGURED_RUNNER = get_runner(settings)
```
[the remark this answers is not in the document]
Comment (kingbuzzman) [reply behave_django/runner.py:10]: @rodrigondec I'm more than happy to close #130 and go forward with yours if you make this change. What do you think?

[the remark this answers is not in the document]
Comment (kingbuzzman) [reply behave_django/runner.py:10]: Never mind, #130 got merged. Not sure if you want to close this or not.

**Comment by bittner** [comment]
@rodrigondec Is there anything in your PR that you see missing in the implementation we have merged with #130? Anything you would like to see integrated from your PR?

1. The test runner can be set via [a CLI option](https://github.com/behave/behave-django/blob/main/behave_django/management/commands/behave.py#L58-L65) elegantly & easily.
2. A test case can be integrated via the custom test runner (following the pattern outlined in the [behave_django.runner](https://github.com/behave/behave-django/blob/main/behave_django/runner.py#L11-L15) module).

**Pull request closed without merging**